openhands-sdk/openhands/sdk/context/agent_context.py

```diff
@@ -48,6 +49,14 @@ class AgentContext(BaseModel):
     user_message_suffix: str | None = Field(
         default=None, description="Optional suffix to append to the user's message."
     )
+    load_user_skills: bool = Field(
+        default=True,
```

Comment:
Let's keep the default to false since we are building a library and we probably don't want to create any surprises for users

```suggestion
        default=False,
```